jwt/grp/types.go

```diff
@@ -18,48 +18,48 @@ type GroupContext struct {
 type Group struct {
 	// Application
 	AppPrefix string
-	// Tenant of the cluster, if is not the same as the tenant prefix (example: ddd of group tnnt_ddd#dev-all-admin)
-	ClusterTenant string
-	// name of the cluster
-	ClusterName string
-	// namespace in the cluster
-	Namespace string
+	// "On behalf"-Tenant, if is not the same as the tenant prefix (example: ddd of group tnnt_ddd#dev-all-admin)
+	OnBehalfTenant string
+	// first scope: for kaas name of the project, for k8s name of the cluster
```

Comment:
First word for documentation should match the variable name. Also I think the documentation is not general enough because this PR seems to make everything more general purpose as before?

Reply:
general purpose in the sense of naming, the structure has not changed - so I think the doc/examples should be ok